Phonebook.Frontend/src/app/pages/dashboard/components/overview/dashboard.component.ts

```diff
@@ -70,39 +72,19 @@ export class DashboardComponent implements OnInit, OnDestroy {
           this.drawerOpen = this.store.selectSnapshot(AppState.recentPeopleDrawer);
         }
       });
-  }
-
-  public changeOrder() {
-    if (this.bookmarkedPersonsSubscriptions) {
-      this.bookmarkedPersonsSubscriptions.unsubscribe();
-    }
-    this.bookmarkedPersonsSubscriptions = this.store
-      .select(BookmarksState.sortedBookmarks)
-      .pipe(map((filterFn) => filterFn(this.favoriteSort)))
-      .subscribe((persons) => {
-        this.bookmarkedPersons = persons;
-      });
-  }
-
-  public entered(e: CdkDragEnter) {
-    if (this.bookmarkedPersons && this.bookmarkedPersons.length === 1) {
-      return;
-    }
-
-    this.lastFrom = e.item.data;
-    this.lastTo = e.container.data;
-  }
-
-  public ended(e: CdkDragEnd) {
-    if (this.bookmarkedPersons && this.bookmarkedPersons.length === 1) {
-      return;
-    }
-    if (this.lastFrom === undefined || this.lastTo === undefined) {
-      return;
-    }
-    moveItemInArray(this.bookmarkedPersons, this.lastFrom, this.lastTo);
-    this.store.dispatch(new UpdateBookmarkOrder(this.bookmarkedPersons));
-    this.cd.detectChanges();
+    this.currentUserService
+      .getCurrentUser()
+      .pipe(untilComponentDestroyed(this))
+      .subscribe(
+        (user) => {
+          if (user != null) {
+            this.currentUser = user;
+          }
+        },
+        (error) => {
+          this.currentUser = null;
+        }
+      );
   }
 
```

Comment:
Please go through this component and remove unused function and vairables (mabe also imports), e.g.: 
- `removeFromBookmarkedPersons`
- `ChangeDetectorRef,` 
- `drawerMode`

Reply:
Maybe the folder should also be renamed in ordert to match up with the component name.